extensions/amp-story/1.0/amp-story-bling-link.css

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-grid-layer amp-story-bling-link {
+  margin: 10px !important;
+  background: rgba(255, 255, 255, 1) !important;
+  border-radius: 50% !important;
+}
+
+amp-story-page[active] amp-story-bling-link {
+  animation: pulse 4s infinite !important;
+}
+
+@-webkit-keyframes pulse {
```

Comment:
You shouldn't need `@-webkit` or `-moz` prefixes, they'll be added at compilation time if needed